src/main/config/config_master.h

```diff
@@ -93,10 +95,12 @@
 #define beeperConfig(x) (&masterConfig.beeperConfig)
 #define sonarConfig(x) (&masterConfig.sonarConfig)
 #define ledStripConfig(x) (&masterConfig.ledStripConfig)
+#define statusLedConfig(x) (&masterConfig.statusLedConfig)
```

Comment:
Should we have this as parameter of the configuration? we also can influence this assignment via IO directly as far as i understand?